nistats/tests/test_make_bids_compliant_localizer_first_level_paradigm_file.py

```diff
@@ -0,0 +1,52 @@
+import os
```

Comment:
I'm wondering why you're not adding that one in test_datasets.py directly. I think I wouldn't like to see 100 files in this directory.